int32 frame_id
float32 x
float32 y
float32 z
float32 roll
float32 pitch
float32 yaw
sensor_msgs/PointCloud2 pointcloud
